tests/test_integration.py

```diff
@@ -31,21 +40,9 @@ def run(conf):
             extensions = ['sphinxcontrib.redoc']
             source_suffix = '.rst'
             master_doc = 'index'
-            redoc = [
-                {
-                    'name': 'Github API (v3)',
-                    'page': 'api/github/index',
-                    'spec': '_specs/github.yml',
-                    {% if opts is not none %}
-                    'opts': {
-                        {% for key, value in opts.items() %}
-                        '{{ key }}': {{ value }},
-                        {% endfor %}
-                    },
-                    {% endif %}
-                },
-            ]
-        ''')).render(opts=conf)
+            redoc = {{ redoc }}
+        ''')).render(redoc=pprint.pformat([defaultconf]))
```

Comment:
Sorry, I maybe missed that, but what's the point in having `pprint.pformat` here?

Reply:
Good catch, it's probably something left over from testing, when I wanted the rendered template to be more human-readable. I'll remove it.